### PR TITLE
My version

### PR DESCRIPTION
**Pull request by appu313**
A new version to the existing collection@casey-collab

### CODE REVIEW

File: english/earlyBird/content.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+1. Becoming an early riser will make you more successful.
+2. Morning people are happier than night owls.
+3. Morning people are often in better shape than night owls.
+4. Waking up early leads to increased productivity.
+5. Possess healthy character traits.
+
+So kudos to you for being an early bird.
+
+Click [here](../marshmallow.md) to go back to the story.
+Click [here](https://www.google.com/) to go to google.
```
Comment (casey-collab) [inline english/earlyBird/content.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/nightOwl/content.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+1. More creative.
+2. Might be more intelligent.
+3. Might have more mental stamina than early birds.
+4. Might get an extra burst of strength.
+5. Might be better at baseball. ;)
+
+So here's a shout out to all the night owls out there.
+
+Click [here](../marshmallow.md) to go back to the story.
+Click [here](https://www.google.com/) to go to google.
```
Comment (casey-collab) [inline english/nightOwl/content.md]: This link is also on the line directly after another link.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by appu313** [comment]
Have inserted blank lines between lines containing links

**Comment by appu313** [comment]
Have inserted blank lines between lines containing links @casey-collab

**Comment by casey-collab** [comment]
Looks great! :fire:

**Pull request merged**